internal/pipe/brew/template.go

```diff
@@ -49,6 +49,7 @@ class {{ .Name }} < Formula
   {{ range $index, $element := . }}
   depends_on "{{ .Name }}"
   {{- if .Type }} => :{{ .Type }}{{- end }}
+  {{- if .Version}} => :{{ .Version }}{{- end }}
```

Comment:
how should it look like if it has both the version and type options set?

this will eval to something like:

```rb
depends_on "foo" => :optional => "v1.2.3"
```

Also I think the version shouldn't be a symbol here...

Reply:
I guess a sensible thing to do here is to use the version if given, and otherwise use the type, if both type and version are given, use just the version... I was looking into homebrew-core and did not find anything using both...

Reply:
btw, I don't remember seeing anything with a version requirement tbh, also searching github rn and also can't find any examples...

https://github.com/search?l=Ruby&q=depends_on&type=Code